src/main/java/htsjdk/samtools/util/SAMRecordPrefetchingIterator.java

```diff
@@ -65,6 +67,7 @@ private void prefetch() {
                 // InterruptedException is expected if the iterator is being closed
                 return;
             } catch (final Throwable t) {
+                t.printStackTrace();
```

Comment:
I think this shouldy be printed in the case of an Error, not in the normal case of an Exception.